src/main/java/be/syntra/devshop/DevshopBack/controllers/ProductController.java

```diff
@@ -116,13 +114,12 @@ public ProductController(
 
     @PostMapping("/searching")
     public ResponseEntity<ProductList> retrieveAllProductsBySearchModel(@RequestBody SearchModelDto searchModelDto) {
-        log.info("retrieveAllProductsBySearchModel -> searchModel{}", searchModelDto);
-        final List<Product> productList = searchService.applySearchModel(
+        final ProductPageAndMinMaxPrice productPageAndMinMaxPrice = searchService.applySearchModel(
                 searchModelMapper.convertToSearchModel(searchModelDto)
         );
```

Comment:
move both this ); and the next, one line up, so they're one the same line as the code they're closing?

Reply:
fixed, dunno why i started doing that ....